Editor/Windows/CinemachineSettings.cs

```diff
@@ -323,12 +323,23 @@ private static void OnGUI()
                 GUILayout.EndScrollView();
             }
 
+            // set label width, so text is not cut for Toggles
+            float originalValue = EditorGUIUtility.labelWidth;
+            EditorGUIUtility.labelWidth = 180f;
```

Comment:
The solution is good, but I don't like this hardcoded number.  It's a problem waiting to happen.
Instead, you should get the number dynamically using GUI.skin.label.CalcSize.  See examples elsewhere in code.  Calc size for both labels and use the max.